packages/modules/promotion/src/services/promotion-module.ts

```diff
@@ -202,12 +206,12 @@ export default class PromotionModuleService<
 
         const campaignBudgetData = {
           id: campaignBudget.id,
-          used: (campaignBudget.used ?? 0) + 1,
+          used: MathBN.add(campaignBudget.used ?? 0, 1),
```

Comment:
@riqwan can you validate if this `+ 1` here is correct?

Reply:
Yes, this is correct